workshops/personal_website/README.md

```diff
@@ -491,34 +488,33 @@ we can share with anyone on the internet! We'll the previously mentioned GitHub
 Pages to do this.
 
 1. Open the terminal by pressing `alt + t` on the keyboard at the same time.
-   Then type in the following commands:
+   Type in the following commands:
   - `git add --all`
   - `git commit -m "Initial commit"`
   - `git push`
-2. GitHub will now ask you for your username and password.
-  - Go ahead and enter your username and then press the enter.
-  - Then enter your password and press enter. _The characters won't show up on
-    the screen, but rest assured, you are still typing._
-3. Now try to view your website by going to
-   `USERNAME.github.io/personal_website` (make sure to replace `USERNAME` with
-   your actual GitHub username)
+2. GitHub will now ask for our username and password.
```

Comment:
This is one instance where I think "your" might be better, since GitHub credentials aren't shared with anyone else.